regression/symtab2gb/multiple_symtabs/test.desc

```diff
@@ -7,4 +7,3 @@ user.json_symtab library.json_symtab
 ^\[2\] file library.adb line \d+ assertion: FAILURE$
 ^VERIFICATION FAILED$
 --
-error
```

Comment:
Why did this line get removed?

Reply:
The test triggered on "error trace", i.e., fails when `--trace` is added.

Reply:
But ... this PR no longer adds a need to compute traces?

Reply:
Yes, but our future selves still want this. But could pull over into separate PR.

Reply:
I'm sorry, I had to convince myself of the problem... Done and now PR'ed as https://github.com/diffblue/cbmc/pull/8270.